src/style.css

```diff
@@ -0,0 +1,146 @@
+/* #region global */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+body,
+p {
+  margin: 0;
+
+  --white-for-card: #fff;
+  --blue-for-card: #00acdc;
+  --black--for-card: #060b35;
+  --grey--for-card: #616070;
+}
+/* #endregion */
+
+/* #region stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
+/* #endregion */
+
+/* #region card */
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card_img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+  margin-left: 3px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+}
+
+.title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--black--for-card);
+}
+
+.code {
+  color: var(--grey--for-card);
+  font-weight: 400;
```

Comment:
that's default value - no need to set it

Reply:
I do not understand, what problem i have with npm test, because all styles I was wrote from link to Figma???